app/views/devise/registrations/new.html.erb

```diff
@@ -0,0 +1,43 @@
+<div class="container mt-5">
+  <div class="user-form">
+    <%= render "devise/shared/error_messages", resource: resource %> </br>
+    <div class="user-form-cont">
+      <h2 class="user-form-title">Sign up</h2>
+      <%= form_for(resource, as: resource_name, url: registration_path(resource_name), html: { class: "form" }) do |f| %>
+        <div class="mb-3">
+          <%= f.label :email, class: "form-label" %>
+          <%= f.email_field :email, autofocus: true, autocomplete: "email", placeholder: "email address", class: "form-control", required: true %>
+        </div>
+        <div class="mb-3">
+          <%= f.label :name, class: "form-label" %>
+          <%= f.text_field :name, autofocus: true, autocomplete: "name", maxlength: 16, placeholder: "full name", class: "form-control", required: true %>
+        </div>
+        <div class="mb-3">
+          <%= f.label :bio, class: "form-label" %>
+          <%= f.text_field :bio, autofocus: true, autocomplete: "bio", placeholder: "bio", class: "form-control", required: true %>
+        </div>
+        <div class="mb-3">
+          <%= f.label :photo, class: "form-label" %>
+          <%= f.text_field :photo, autofocus: true, autocomplete: "photo", placeholder: "profile picture url", class: "form-control" %>
+          <%= f.hidden_field :photo, value: "https://previews.123rf.com/images/valentint/valentint1406/valentint140601650/29505264-perfil-de-usuario-icono-bot%C3%B3n-met%C3%A1lico-de-internet-sobre-fondo-blanco.jpg" %>
+          <small class="form-text text-muted">(leave it blank to use default picture)</small>
+        </div>
+        <div class="mb-3">
+          <%= f.label :password, class: "form-label" %>
+          <%= f.password_field :password, autocomplete: "new-password", placeholder: "new password", class: "form-control", required: true %>
+          <% if @minimum_password_length %>
+            <small class="form-text text-muted"><%= @minimum_password_length %> characters minimum</small>
+          <% end %>
+        </div>
+        <div class="mb-3">
+          <%= f.label :password_confirmation, class: "form-label" %>
+          <%= f.password_field :password_confirmation, autocomplete: "new-password", placeholder: "confirm new password", class: "form-control", required: true %>
+        </div>
+        <%= f.submit "Sign up", class: 'btn btn-primary' %>
+      <% end %>
+      <div class="mt-3 shared-links text-center">
+        <%= link_to "Go back", users_path, class: 'btn btn-secondary shared-link go-back' %>
+      </div>
+    </div>
+  </div>
+</div>
```

Comment:
- [ ] Nice job so far for the awesome login and signup layout 👌This activity, however, requires the modification of the views in _devise_ for the registration also matches the styling of the [provided wireframe(s)](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/sneak_peek.md). Kindly ensure that the form (inputs and buttons) stretches across the screen and is centered accordingly for the **Signup** page, as seen in the [wireframe](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/sneak_peek.md):

**Your Signup view:**        |  **Your login view (Great):**        |  **Wireframe login view (Expected):**
:-------------------------:|:-------------------------:|:-------------------------:
<img width="800" alt="Home Page" src="https://github.com/sergio-usma/rails_blog_pj/assets/12745474/75b5fb42-dc9d-4d23-90db-e2ed892916d8"> | <img width="800" alt="Home Page" src="https://github.com/sergio-usma/rails_blog_pj/assets/12745474/55d7416c-d83d-4b8a-88d5-e1a1ed92e984"> | <img width="800" alt="Home Page" src="https://user-images.githubusercontent.com/12745474/192667669-04020765-a648-4c50-ab45-7d68eda203e7.png">